### PR TITLE
Add Teams webhook support

### PR DESCRIPTION
**Pull request by noklam**
🚪✊🚪✊
Add Microsoft Teams sender, it is not too different from slack as it is based on webhook.

### CODE REVIEW

File: knockknock/teams_hook_sender.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+from typing import List
```
Comment (VictorSanh) [inline knockknock/teams_hook_sender.py]: It seems like this file is a duplicate of `teams_sender.py`. Do we need it?

**Comment by VictorSanh** [comment]
Sweet! Thank you @noklam!
I am not really familiar with Microsoft Teams but it looks good to me!

Do you mind adding a few lines in the README following the other examples?

**Comment by VictorSanh** [comment]
NB: I like the emojis for the PR by the way ;)

**Comment by noklam** [comment]
@VictorSanh Thx!

**Comment by VictorSanh** [comment]
Thanks for the contrib, i'll update the pip package!

**Pull request merged**